website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

```diff
@@ -0,0 +1,189 @@
+4. Save and commit your changes.
+5. Restart the IDE.
+6. Test it out and happy linting!
+
+:::tip Configure dbtonic linting rules
+
+Use the following code example to incorporate well-written dbt code (or dbtonic) to your linting:
+
+<details>
+<summary>dbtonic config code example provided by dbt Labs</summary>
+
+```
+[sqlfluff]
+templater = dbt
+# This change (from jinja to dbt templater) will make linting slower
+# because linting will first compile dbt code into data warehouse code.
+runaway_limit = 10
+max_line_length = 80
+indent_unit = space
+
+[sqlfluff:indentation]
+tab_space_size = 4
+
+[sqlfluff:layout:type:comma]
+spacing_before = touch
+line_position = trailing
+
+[sqlfluff:rules:capitalisation.keywords] 
+capitalisation_policy = lower
+
+[sqlfluff:rules:aliasing.table]
+aliasing = explicit
+
+[sqlfluff:rules:aliasing.column]
+aliasing = explicit
+
+[sqlfluff:rules:aliasing.expression]
+allow_scalar = False
+
+[sqlfluff:rules:capitalisation.identifiers]
+extended_capitalisation_policy = lower
+
+[sqlfluff:rules:capitalisation.functions]
+capitalisation_policy = lower
+
+[sqlfluff:rules:capitalisation.literals]
+capitalisation_policy = lower
+
+[sqlfluff:rules:ambiguous.column_references]  # Number in group by
+group_by_and_order_by_style = implicit
+```
+</details>
+:::
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-sqlfluff-config.jpg" width="95%" title="Customize linting by configuring your own linting code rules, including dbtonic linting/styling."/>
+
+## Format
+
+In the dbt Cloud IDE, you can format your code to match style guides with a click of a button. The IDE integrates with formatters like sqlfmt, Prettier, and Black to automatically format code on five different file types &mdash; SQL, YAML, Markdown, Python, and JSON:
+
+- SQL &mdash; Format with [sqlfmt](http://sqlfmt.com/), which provides one way to format your dbt SQL and Jinja.
+- YAML, Markdown, and JSON &mdash; Format with [Prettier](https://prettier.io/). 
+- Python &mdash; Format with [Black](https://black.readthedocs.io/en/latest/).
+
+The Cloud IDE formatting integrations take care of manual tasks like code formatting, enabling you to focus on creating quality data models, collaborating, and driving impactful results.
+
+### Format SQL
+
+To format your SQL code, dbt Cloud integrates with [sqlfmt](http://sqlfmt.com/), which is an uncompromising SQL query formatter that provides one way to format the SQL query and Jinja.
+
+To enable formatting:
+
+1. Make sure you open a `.sql` file.
+2. Click on the **`</> Config`** button on the right side of the console. (Make sure you're in a .sql file).
+3. In the code quality tool config pop up, you have the option to select sqlfluff or sqlfmt. 
+4. To format your code, select the **sqlfmt** radio button. (Use sqlfluff to [lint](#linting) your code).
+5. Once you've selected the **sqlfmt** radio button, go to the console section (located below the **File editor**) to select the **Format** button. 
```

Comment:
Does "console section" mean different things in different steps? It's a little vague and might benefit from a more specific language, if possible.